packages/desktop-client/src/components/settings/Experimental.js

```diff
@@ -34,7 +34,13 @@ export default function ExperimentalFeatures({ prefs, savePrefs }) {
               <View>
                 Budget mode toggle
                 {disabled && (
-                  <Text style={{ color: colors.r3, fontWeight: 500 }}>
+                  <Text
+                    style={{
+                      backgroundColor: colors.errorBackground,
```

Comment:
Agree it looks out of place. For dark theme I try and keep error text and background together, otherwise visibility in certain scenarios suffers. I added some padding but it still isn't perfect.

Reply:
Would it work to either have the text color in dark mode work with both the regular background and the error background? Or have 2 different error text colors for error messages w/ background and without?